tenzir/integration/data/reference/expression/test_ip_in_subnet/step_00.ref

```diff
@@ -2,4 +2,24 @@
 {"x": "1.2.3.4", "y": "4.5.0.0/16", "z": false}
 {"x": "1.2.3.4", "y": null, "z": null}
 {"x": null, "y": "4.5.0.0/16", "z": null}
-{"x": null, "y": null, "z": null}
\ No newline at end of file
+{"x": null, "y": null, "z": null}
+warning: binary operator `in` not implemented for `ip` and `null`
+ --> /dev/stdin:6:5
+  |
+6 | z = x in y
+  |     ~~~~~~ 
+  |
+
+warning: binary operator `in` not implemented for `null` and `subnet`
+ --> /dev/stdin:6:5
+  |
+6 | z = x in y
+  |     ~~~~~~ 
+  |
+
+warning: binary operator `in` not implemented for `null` and `null`
+ --> /dev/stdin:6:5
+  |
+6 | z = x in y
+  |     ~~~~~~ 
+  |
```

Comment:
I assume this is one of the tests that relies on the type unification. What do you think should we do with this?